src/components/button/Button.css

```diff
@@ -1,11 +1,11 @@
 .main-button {
-  background-color: #55198b;
-  border: solid 1px #55198b;
+  background-color: #6c63ff;
```

Comment:
The background colour of Button is changed which one is good? `#55198b` or `#6c63ff`? @kartik918

Reply:
I prefer the old scheme since it's more contrasting to white, and light on light doesn't look too good I think. I want to know your opinion too.

Reply:
According to me, both the things look good. Lets ask @saadpasta .

Reply:
Both look good. But old has a good contrast to white.

Reply:
So which one can we use @saadpasta and @kartik918?

Reply:
Old hex codes like Saad said i.e the dark purple color would be awesome.